.changeset/heavy-terms-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"vercel": patch
```

Comment:
This is probably technically a minor (introduces new features), but not blocking.